tasks/branch.js

```diff
@@ -0,0 +1,36 @@
+
+'use strict';
+
+module.exports = function(grunt) {
+
+  var templateHelper = require('./lib/templateHelper').init(grunt);
+  var _ = require('underscore');
+
+  function branchAppTemplates(filepath, branch) {
+    grunt.config.set('gitcheckout', {});
+
+    var globalJson = grunt.file.readJSON(filepath);
+
+    templateHelper.walkObjects(globalJson.show, function (obj) {
+      var prefix = 'gitcheckout.' + obj.id + '.options';
+      if (_.has(obj, 'repoUrl') && _.has(obj, 'repoBranch')) {
+        grunt.config.set(prefix + '.branch', branch);
+        grunt.config.set(prefix + '.create', true);
+        grunt.config.set(prefix + '.cwd', 'tmp/' + obj.id);
+      }
+    });
+    if (_.isEmpty(grunt.config.get('gitcheckout'))) {
+      grunt.log.ok("No git apps defined in " + filepath);
+    } else {
+      grunt.task.run('gitcheckout')
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)